e2e/e2etests/test_ton_withdrawals.go

```diff
@@ -0,0 +1,96 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"cosmossdk.io/math"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	toncontracts "github.com/zeta-chain/node/pkg/contracts/ton"
+	"github.com/zeta-chain/node/zetaclient/chains/ton/liteapi"
+)
+
+func TestTONWithdraw(r *runner.E2ERunner, args []string) {
+	// ARRANGE
+	require.Len(r, args, 1)
+
+	// Given a deployer
+	_, deployer := r.Ctx, r.TONDeployer
+
+	// That donates 100 TON to some zEVM sender
+	zevmSender := r.ZEVMAuth.From
+
+	_, err := r.TONDeposit(&deployer.Wallet, toncontracts.Coins(100), zevmSender)
+	require.NoError(r, err)
+
+	// Given his ZRC-20 balance
+	senderZRC20BalanceBefore, err := r.TONZRC20.BalanceOf(&bind.CallOpts{}, zevmSender)
+	require.NoError(r, err)
+	r.Logger.Print("zEVM sender's ZRC20 TON balance before withdraw: %d", senderZRC20BalanceBefore)
+
+	// Given another TON wallet
+	tonRecipient, err := deployer.CreateWallet(r.Ctx, toncontracts.Coins(1))
+	require.NoError(r, err)
+
+	tonRecipientBalanceBefore, err := deployer.GetBalanceOf(r.Ctx, tonRecipient.GetAddress())
+	require.NoError(r, err)
+
+	r.Logger.Print("Recipient's TON balance before withdrawal: %s", toncontracts.FormatCoins(tonRecipientBalanceBefore))
+
+	// Given amount to withdraw (and approved amount in TON ZRC20 to cover the gas fee)
+	amount := parseUint(r, args[0])
```

Comment:
Is there any limitation on the minimum amount being withdrawn? For say, if a user withdraw `1e-9` to a brand new empty address in TON, would that be okay or be rejected (by minimum balance requirement)?

Reply:
it would be accepted and funds will be moved to an `uninit` TON account. That's expected

Reply:
okay